### ISSUE TITLE
Remove the little crown symbol next to the host name during gameplay

### ISSUE THREAD

**Issue by thefloridianboy** [ipodtouch0218/NSMB-MarioVsLuigi#158]
idk its just distracting to see it

**Comment by Superyoshi94** [comment]
I don't get the point of doing that, but it would be cool if there was a secondary crown for the person with the most stars.

**Comment by Superyoshi94** [comment]
> I don't get the point of doing that, but it would be cool if there was a secondary crown for the person with the most stars.

But also there really isn't any point in doing this either.

**Comment by Amy54Desu** [comment]
> > I don't get the point of doing that, but it would be cool if there was a secondary crown for the person with the most stars.
> 
> But also there really isn't any point in doing this either.

I disagree, a crown would make it more obvious they have the most stars especially with leaderboard off

**Comment by ipodtouch0218** [comment]
This was changed in v2.0.0.0, closing as completed.
